gdx/src/com/badlogic/gdx/graphics/FPSLogger.java

```diff
@@ -39,11 +39,19 @@ public FPSLogger (int bound) {
 
 	/** Logs the current frames per second to the console. */
 	public void log () {
+		log(1);
+	}
+	
+	public void log(int interval) {

```

Comment:
This overload of log() should be documented, including the interval parameter. I also strongly think interval should be a double (not a float; these numbers are way outside precise float range), because it allows users to refresh frame rate more often than once per second.